backend/src/main/resources/application.yaml

```diff
@@ -10,4 +10,4 @@ spring:
     show-sql: false
     database-platform: org.hibernate.dialect.MySQL8Dialect
     hibernate:
-      ddl-auto: update
+      ddl-auto: create
```

Comment:
Ez biztos? Create: "Create the schema and destroy previous data". Egy idő után ki is kell majd kapcsolni, ha ezt deployolod az élesre, akkor induláskor kiüríti a DB-det. Nem tudok olyan szituációt elképzelni, amikor egy PROD-ot így akarok kezelni. A local/personal application-xx.yaml-ben ez lehet értelmes, itt nem.

Reply:
ez véletlen maradt bent kicseréltem